Add Typelevel Summit 2023 event (#463)
typelevel/typelevel.github.com: collections/_events/2023-10-26-summit-nescala.md

---
layout: event

title: "Typelevel Summit NEScala"
short_title: "Typelevel Summit"
date_string: "October 26, 2023"
location: "Online"

> description: "One day of recorded talks and one day of unconference, co-located with NE Scala."

Reviewer: ```suggestion
description: "One day of talks and one day of unconference, co-located with NE Scala."
```
Calling it "recorded talks" makes it sounds like the talks will all be pre-recorded, which I don't think is true -- while we're going to allow for pre-recorded, I expect most to be live, and for some to have Q&A.  Tweaked the section below to clarify (I think).

---

## About the Summit

> The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of recorded talks and one day of (shared) unconference.

Reviewer: ```suggestion
The tenth Typelevel Summit will once again be co-hosted with the [Northeast Scala Symposium](http://www.nescala.org/) virtually, with one day of talks and socializing online and one day of unconference.  Talks will be recorded and available online afterwards.
```

Reviewer 2: maybe "discussion and socializing"?

Reviewer 2: or "one of day of talks and one day of unconference, with discussion and socializing online throughout"? idk

Reviewer: I like that second version!

The schedule for this year is as follows:

* October 26 (Thursday): Typelevel Summit

> * October 27 (Friday): NE Scala 2020

Reviewer: ```suggestion
* October 27 (Friday): NE Scala 2023
```
Do you know where you copied this from?  Sounds like we might need to fix something somewhere.
